pkg/spec/spec.go

```diff
@@ -72,6 +72,7 @@ type ConfigMapMod struct {
 
 type PodSpecMod struct {
 	Containers     []Container `json:"containers,omitempty"`
+	InitContainers []Container `json:"initContainers,omitempty"`
```

Comment:
@kadel but we have things in our `Container` struct that enhance it, like `Health` and `envFrom` that won't show up in `InitContainers`.